### PR TITLE
Add support for rainbow_parentheses.vim

### PR DESCRIPTION
**Pull request by bbakersmith**
1989-rainbow-parentheses

### CODE REVIEW

File: colors/1989.vim [hunk 1]
```diff
@@ -166,3 +166,11 @@ call <SID>set_hi("markdownH5", s:light_blue, s:none, "NONE")
 call <SID>set_hi("markdownH6", s:light_blue, s:none, "NONE")
 call <SID>set_hi("markdownHeadingDelimiter", s:light_blue, s:none, "NONE")
 call <SID>set_hi("markdownRule", s:light_blue, s:none, "NONE")
+
+" https://github.com/kien/rainbow_parentheses.vim
+let g:rbpt_colorpairs = [
```
Comment (gabebw) [inline colors/1989.vim]: Should this be wrapped in `if exists( g:rbpt_colorpairs)` so that it doesn't overwrite a user's settings?

Comment (bbakersmith) [inline colors/1989.vim]: Good point, updated to not overwrite rbpt_colorpairs if already defined in vimrc.

Comment (bbakersmith) [inline colors/1989.vim]: Actually @gabebw I tested a bit further and if I have g:rbpt_colorpairs set in my vimrc it overrides the colorscheme. I reverted the commit with exists(), do you see different behavior?

Comment (gabebw) [inline colors/1989.vim]: Ah, I'm using https://github.com/luochen1990/rainbow, which doesn't use the same variable, so you can just ignore me 🐱

**Comment by sonjapeterson** [comment]
Hi! Thanks for the PR! I tested it out and it does seem to override user defined rgbt_colorpairs if you set your colorscheme after defining the colorpairs in your vimrc, like this:

```
let g:rbpt_colorpairs = [
    \ ['blue',       'RoyalBlue3'],
    \ ['green',    'SeaGreen3'],
    \ ['gray',    'DarkOrchid3'],
    \ ['brown',   'firebrick3'],
    \ ]

colorscheme 1989
```

Overriding someone's user settings for a different plugin via a colorscheme seems a bit surprising so I'd rather you wrapped it in a `if !exists("g:rbpt_colorpairs")`.

Thanks again!

**Comment by bbakersmith** [comment]
Ah yes, you're right, thanks for catching that. PR updated to respect user's rbpt_colorpairs.

**Comment by sonjapeterson** [comment]
Great, thank you!

**Pull request merged**